lib/stripe/stripe_object.rb

```diff
@@ -184,5 +184,72 @@ def method_missing(name, *args)
     def respond_to_missing?(symbol, include_private = false)
       @values && @values.has_key?(symbol) || super
     end
+
+    protected
+
+    def serialize_nested(key)
```

Comment:
The name here feels a bit unclear . Maybe `serialize_field` or `serialize_value` instead?